src/utils/csvParser/parser.test.ts

```diff
@@ -0,0 +1,1544 @@
+/* eslint-disable @typescript-eslint/no-loss-of-precision */
+import {expect, jest} from '@jest/globals'
```

Comment:
is this import necessary or can it be done away with? it doesn't look like any of our other tests import `expect` or `jest`

Reply:
you're right. Other unit tests in the code base don't import jest 🤔. But removing the import throws a lot of errors like this: 
<img width="913" alt="jest_error" src="https://user-images.githubusercontent.com/66275100/195628233-810d30b1-f31f-4e39-a7d8-3fad3da2e9e3.png">
I can't think of why this test file requires an import while others don't. I'll look more into this

Reply:
I wonder if this is the first test to be created after we upgraded jest